src/Importify/Resolution.hs

```diff
@@ -50,3 +64,15 @@ resolveOneModule m =
         symbolsEnv    = resolve [clearedModule] mempty
         symbols       = concat $ M.elems symbolsEnv
     in symbols
+
+collectUsedQuals :: [ImportDecl SrcSpanInfo] -> [Scoped SrcSpanInfo] -> [ModuleName SrcSpanInfo]
```

Comment:
This function is exported thus should have _Haddock_ documentation what it does and if required -- why.